dev/archery/archery/release.py

```diff
@@ -39,16 +39,19 @@ class Version(SemVer):
 
     __slots__ = SemVer.__slots__ + ('released', 'release_date')
```

Comment:
Aside: slots shouldn't need to be duplicated like this

Reply:
Thanks.
I've removed it.